src/api/item.ts

```diff
@@ -43,9 +43,14 @@ export const getItems = async (
 ): Promise<ResultOf<Item>> =>
   axios.get(`${API_HOST}/${buildGetItemsRoute(ids)}`).then(({ data }) => data);
 
-export const getOwnItems = async ({ API_HOST }: QueryClientConfig) =>
+export const getOwnItems = async (
+  { API_HOST }: QueryClientConfig,
+  { page = 1, all = false, name = '', limit }: OwnItemsQuery,
```

Comment:
As mentioned in the backend PR, I would not have `all`, and separate `name` from the pagination args. 
Shouldn't `limit` have a default as well?